README.md

```diff
@@ -1 +1,148 @@
-# Air-Quality
\ No newline at end of file
+<a name="readme-top"></a>
+
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Air Quality ](#-air-quality-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+# 📖 Air Quality <a name="about-project"></a>
+
+> The website displays a list of countries that are fetched from rest world API. Once the country is selected, the respective cities are loaded. On selecting the city, its respective air data is displayed on a table over a short period of time.
+
+**[Air Quality]** is:
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> The project is built using.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+    <li><a href="https://react-bootstrap.github.io/">React Bootstrap</a></li>
+    <li><a href="https://jsx.github.io/">JSX</a></li>
+    <li><a href="https://www.w3.org/Style/CSS/Overview.en.html">CSS</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://nodejs.org/en/">Node.js</a></li>
+    <li><a href="https://babeljs.io/">Babel</a></li>
+    <li><a href="https://jestjs.io/">Jest</a></li>
+  </ul>
+</details>
+
+<br>
+
+### Key Features <a name="key-features"></a>
+
+> The key features of the application are:
+
+- **[Fetches data from API]**
+- **[Interactive buttons to load specific cities and air data when pressed]**
+- **[Styling done using React Bootstrap]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Link to deployed project is:
+
+- [Live Demo Link](https://space-x1234.netlify.app/)
```

Comment:
- [ ] Your project should be deployed properly to Heroku or Netify. The Home page and the detail page should work as expected. A link to the deployed project should be present in your README. This link does not contain the expected project. Kindly fix :wrench: . This requirement if essential to the approval of this project.

Reply:
Thank you for your time, I have updated the README.md with the correct link.